### PR TITLE
updated ala-collectory related yml files

### PR DESCRIPTION
**Pull request by linaizhong**
I have updated the yml files for deploy ala-collectory project.

### CODE REVIEW

**Comment by djtfmartin** [comment]
Please remove all the images. These data shouldn't be part of the install.

File: ansible/roles/collectory/tasks/main.yml [hunk 1]
```diff
@@ -61,16 +65,17 @@
     - properties
 
 - name: copy all template configs
-  template: src={{item}} dest={{data_dir}}/collectory/{{item}}
+  template: src={{item}} dest={{data_dir}}/ala-collectory/{{item}}
   with_items:
-    - config/collectory-config.properties 
+    - config/ala-collectory-config.properties
+    - config/connection-profiles.json
```
Comment (m-r-c) [inline ansible/roles/collectory/tasks/main.yml]: roles/collectory/templates/connection-profiles.json does not exist?

File: ansible/roles/collectory/tasks/main.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@
   copy: src={{item}} dest=/tmp
   with_items:
     - collectory-sequences.sql
+    - collectory-schema.sql
   tags:
```
Comment (m-r-c) [inline ansible/roles/collectory/tasks/main.yml]: Please add a 'collectory' tag to each task - this lets us run a cut down version of the playbook by specifying the tag

File: ansible/roles/collectory/files/collectory-schema.sql [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+alter table data_resource add index FKE040503A1605ED4 (institution_id), add constraint FKE040503A1605ED4 foreign key (institution_id) references institution (id);
+alter table provider_map add index FKE70A7A0EB01EB1A0 (collection_id), add constraint FKE70A7A0EB01EB1A0 foreign key (collection_id) references collection (id);
+alter table provider_map_provider_code add index FK13BB0B0A2B882E69 (provider_map_collection_codes_id), add constraint FK13BB0B0A2B882E69 foreign key (provider_map_collection_codes_id) references provider_map (id);
+alter table provider_map_provider_code add index FK13BB0B0A195335D1 (provider_map_institution_codes_id), add constraint FK13BB0B0A195335D1 foreign key (provider_map_institution_codes_id) references provider_map (id);
+alter table provider_map_provider_code add index FK13BB0B0A79EFEF03 (provider_code_id), add constraint FK13BB0B0A79EFEF03 foreign key (provider_code_id) references provider_code (id);
+create index uid_idx on temp_data_resource (uid);
+CREATE TABLE `sequence` (`id` bigint(20) NOT NULL,`next_id` bigint(20) NOT NULL,`name` varchar(45) NOT NULL,`prefix` varchar(5) NOT NULL, PRIMARY KEY (`id`));
```
Comment (m-r-c) [inline ansible/roles/collectory/files/collectory-schema.sql]: collectory-sequences.sql creates this table. This causes an error at run time.

File: ansible/roles/collectory/tasks/main.yml (hunk 1 above)
Comment (m-r-c) [inline ansible/roles/collectory/tasks/main.yml]: Looks like it exists in roles/collectory/files/data/config. Is it a template or a static file? If it's a template, please move it to the templates directory. If it is static, please remove the reference at line 71.

File: ansible/roles/collectory/tasks/main.yml [hunk 4]
```diff
@@ -32,28 +34,30 @@
...
+  file: path={{data_dir}}/ala-collectory owner={{tomcat_user}} group={{tomcat_user}}  state=directory
   tags:
     - properties
 
 - name: ensure target directories exist [data subdirectories etc.]
   file: path={{item}} state=directory
   with_items:
-    - "{{data_dir}}/collectory/upload/tmp"
-    - "{{data_dir}}/collectory/taxa/data"
+    - "{{data_dir}}/ala-collectory/upload/tmp"
```
Comment (m-r-c) [inline ansible/roles/collectory/tasks/main.yml]: You can combine this task with the previous one with something like this:
```
 - name: ensure target directories exis
   file: path={{item}} state=directory owner={{tomcat_user}} group={{tomcat_user}}
   with_items:
     - "{{data_dir}}/ala-collectory
     - "{{data_dir}}/ala-collectory/upload/tmp"
     - "{{data_dir}}/ala-collectory/taxa/data"
   tags:
     - properties
     - collectory
```

File: ansible/roles/collectory/tasks/main.yml [hunk 5]
```diff
@@ -32,28 +34,30 @@
...
+    - "{{data_dir}}/ala-collectory/taxa/data"
   tags:
     - properties
 
 - name: copy all data assets [SOLR, lucene etc]
-  copy: src={{item}} dest={{data_dir}}/collectory owner={{tomcat_user}} group={{tomcat_user}}
+  copy: src={{item}} dest={{data_dir}}/ala-collectory owner={{tomcat_user}} group={{tomcat_user}} recurse=true
```
Comment (m-r-c) [inline ansible/roles/collectory/tasks/main.yml]: recurse is not supported on the copy task

**Comment by m-r-c** [comment]
Please test this script against a vagrant VM (using the inventory from ansible/inventories/vagrant/collectory-vagrant).

**Comment by m-r-c** [comment]
These changes only support deployment of the 'ala-collectory' implementation of the collectory plugin. To support other implementations (e.g. generic-collectory), we'll need to make some changes to the directory names, and the playbook, similar to the approach taken for the biocache-hub role and the appd playbook.

**Pull request merged**